com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/FinalPass.shader

```diff
@@ -95,15 +102,18 @@ Shader "Hidden/HDRP/FinalPass"
 
             float2 positionNDC = input.texcoord;
             uint2 positionSS = input.texcoord * _ScreenSize.xy;
+            uint2 scaledPositionSS = ((input.texcoord.xy * _UVTransform.xy) + _UVTransform.zw) * _ViewPortSize.xy;
 
             // Flip logic
             positionSS = positionSS * _UVTransform.xy + _UVTransform.zw * (_ScreenSize.xy - 1.0);
             positionNDC = positionNDC * _UVTransform.xy + _UVTransform.zw;
 
             #ifdef CATMULL_ROM_4
             CTYPE outColor = UpscaledResult(positionNDC.xy);
+            #elif defined(RCAS)
+            CTYPE outColor = ApplyRCAS(scaledPositionSS);
```

Comment:
ApplyRCAS returns a float3, yet you have a CTYPE. This means that you could potentially be setting alpha 0 and causing a shader compiler warning (float3 casted into float4).

Notice how the other passes use the CTYPE_SWIZZE to address this. Can you dig in a bit into support for alpha? I think this might break this pass.

Reply:
I added a new commit that adds some comments explaining how this is handled:

https://github.com/Unity-Technologies/Graphics/blob/d2d314354490699d5f0f67e43fd301026869c394/com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/FinalPass.shader#L41-L48

Any idea if this functionality tested in the DRS tests?